README.md

```diff
@@ -353,6 +355,19 @@ To run the tests:
 
     pytest
 
+To run CI checks locally:
+
+    pre-commit run --all-files
+
+To format source code that fails `isort` or `black` checks:
+
+    isort --skip env . --skip pyinstrument/vendor .
+    black .
+
+To diagnose why `pyright` checks are failing:
+
+    PYRIGHT_PYTHON_FORCE_VERSION=1.1.134 pyright
+
```

Comment:
```suggestion
To run linting checks locally:

    pre-commit run --all-files

Some of the pre-commit checks, like `isort` or `black`, will auto-fix the problems they find. So if the above command returns an error, try running it again, it might succeed the second time :)

Running all the checks can be slow, you can also run checks individually, e.g. to format source code that fails `isort` or `black` checks:

    pre-commit run --all-files isort
    pre-commit run --all-files black

To diagnose why `pyright` checks are failing:

    pre-commit run --all-files pyright
```

Reply:
Made this change in [`235795ad`](https://github.com/joerick/pyinstrument/pull/161/commits/235795ad4551635819ad04719b19b6160dcc050f); the only additional edit I made to the text above was changing the substring "slow, you" to "slow, so you".